cibuildwheel/functools_cached_property_38.py

```diff
@@ -21,10 +21,8 @@ def __set_name__(self, owner: type[Any], name: str) -> None:
         if self.attrname is None:
             self.attrname = name
         elif name != self.attrname:
-            raise TypeError(
-                "Cannot assign the same cached_property to two different names "
-                f"({self.attrname!r} and {name!r})."
-            )
+            msg = "Cannot assign the same cached_property to two different names ({self.attrname!r} and {name!r})."
```

Comment:
```suggestion
            msg = f"Cannot assign the same cached_property to two different names ({self.attrname!r} and {name!r})."
```

Not sure why GitHub won't let me see @joerick's commit in the files tab so I can batch it...